cmd/ulimit_unix.go

```diff
@@ -0,0 +1,45 @@
+//go:build darwin || linux
+// +build darwin linux
+
+package cmd
+
+import (
+	"fmt"
+	"syscall"
+
+	zerolog "github.com/rs/zerolog/log"
+)
+
+const ulimitUnix uint64 = 16384
+
+func init() {
+	fileDescriptorF = checkAndSetUlimitUnix
```

Comment:
why we can call `checkAndSetUlimitUnix` directly ? This should also remove the need for this [check](https://github.com/cloudquery/cloudquery/pull/416/files#diff-ab967ab1a2f3a1b769106eeb7bfe892ef0e81d1d27811fa15be08e6749feee1fR145) unless Im missing some edge-case

Reply:
It should fail on windows if I'm not completely mistaken. Since `syscall` is not available.